python/mxnet/metric.py

```diff
@@ -150,7 +227,30 @@ def update(self, labels, preds):
             self.num_inst += len(pred_label.flat)
 
 class TopKAccuracy(EvalMetric):
-    """Calculate top k predictions accuracy."""
+    """Calculate top k predictions accuracy.
+
+    `TopKAccuracy` computes accuracy slightly different than `Accuracy`.
```

Comment:
`TopKAccuracy` differs from `Accuracy` in that it considers the prediction to be `True` as long as the ground truth label is in the top K predicated labels.